ts-client/tsconfig.json

```diff
@@ -1,11 +1,25 @@
 {
   "compilerOptions": {
-    "types": ["mocha", "chai"],
-    "typeRoots": ["./node_modules/@types"],
-    "lib": ["es2015"],
-    "module": "commonjs",
-    "target": "es6",
+    "types": [
+      "mocha",
+      "chai"
+    ],
+    "target": "es2020",
+    "skipLibCheck": true,
+    "allowJs": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
     "esModuleInterop": true,
-    "resolveJsonModule": true
-  }
+    "noImplicitAny": false,
+    "noImplicitThis": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "rootDir": "src",
+    "outDir": "./lib",
+    "declaration": true
+  },
+  "include": ["**/*.ts"],
+  "exclude": ["node_modules", "lib", "mango-client-v3", "tests"],
+  "types": "./types/*.d.ts"
```

Comment:
1. do we need to target es2020? I don't mind, just wondering if there will be compatibility issue
2. allowJs, we don't need this?